community/kernel/src/main/java/org/neo4j/kernel/api/txstate/TxStateVisitor.java

```diff
@@ -38,301 +38,240 @@
  */
 public interface TxStateVisitor
 {
-    void visitCreatedNode( long id );
+    default void visitCreatedNode( long id ) {}
```

Comment:
I am unsure this is a good idea. I would like the compiler to report errors if there are missing method in the class implementing this interface.